ci/edmtool.py

```diff
@@ -228,7 +228,7 @@ def install(runtime, toolkit, pillow, environment, source):
     # dependencies or we risk testing against an released enable.
     install_local = (
         "edm run -e {environment} -- "
-        "pip install --force-reinstall --no-dependencies -e ."
+        "pip install --force-reinstall --no-dependencies " + ROOT
```

Comment:
I'm not exactly sure why, but I'm not seeing the test failure anymore

Reply:
Previously the test failure reflects the issue that I ran into 😅 : the examples files are moved inside `demo` subdirectory but the `package_data` has not been updated to say `demo/enable/*` etc.
Using `pip install -e .` "fixes" the test failure because it actually makes the package available from source: the package_data is not really relevant in that setup.

After https://github.com/enthought/enable/pull/494/commits/7cecefbdf62fcd26ff5be1a40dcae4cb5f7d8dc8, the `package_data` is fixed to reference the `demo` subdirectory, and that truly fixes the test.

This is in fact one instance where it is valuable to have a CI job where we don't install the source with editable mode. It tests the installed environment more truthfully.

Reply:
Ahhh I see, that makes sense now, thank you!